src/services/config.js

```diff
@@ -129,9 +129,13 @@ class ConfigData {
   }
 
   removeProfile(profileToRemove) {
-    this.projects = this.projects.filter((profile) => {
```

Comment:
Regarding [L154](https://github.com/twilio/twilio-cli-core/pull/126/files#diff-69b5d67d841057efec105a48299840e180ffc94f0da052f105dd16f5465d6a9cR154), what is the better way to remove the keys form keytar? 
1. Do we need to directly call [`secureStorage`](https://github.com/twilio/twilio-cli-core/blob/main/src/services/secure-storage.js#L47) method?
2. Do we need to let user know that we are deleting the key in keytar and updating it in config file or just log the status of key deletion from keytar?

Reply:
I think the logic for dealing with `secureStorage` should be moved here so it can be easily updated.

Reply:
But when I am trying to use `secureStorage` in this file, it's leading to [circular dependency](https://github.com/twilio/twilio-cli-core/blob/release-feature-branch/src/services/secure-storage.js#L1).

Reply:
Wouldn't they be removed as part of deleteLocalKey already even before removeProfile is called?

Reply:
Yes, it will be removed in case of `profile:remove` cmd, but there is another case where we are updating the existing profile, if the profile found in projects we are removing it from [`projects`](https://github.com/twilio/twilio-cli-core/blob/release-feature-branch/src/services/config.js#L151) (but API keys we are not removing from keytar) and [updating](https://github.com/twilio/twilio-cli-core/blob/release-feature-branch/src/services/config.js#L155) in `profiles` key. This flow will come in case of `profile:create existingProfileName`.

Reply:
ok, we have a separate ticket for the update case. Perhaps we could call remove and create there for existing profiles, but let's not put that mix that with this